app/components/scrollable_section_list/virtualized_section_list.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+/* eslint-disable */
+
+import React from 'react';
+import {VirtualizedList} from 'react-native';
+import VirtualizedSectionList from 'react-native/Libraries/Lists/VirtualizedSectionList';
+
+export default class VirtualizedScrollableSectionList extends VirtualizedSectionList {
```

Comment:
What are we actually doing here to make this scrollable? This also feels like we're using a more-internal part of RN that will break because Facebook likes doing that